docs/using-profiles.md

```diff
@@ -0,0 +1,43 @@
+(using-lint-profiles)=
+
+# Applying lint profiles
+
+Ansible-lint profiles allow content creators to progressively improve the quality of Ansible playbooks, roles, and collections.
```

Comment:
@ansible/devtools "s/content creator/content developer" ?

Reply:
@ssbarnea not sure if you spotted this or have a strong feeling. I'm leaving as-is to identify the "content creator" role. Cheers.

Reply:
Nope, please keep the old URL (filename), I am not keen on altering the URLs as it requires extra work. Better to improve the docs without changing urls.

Reply:
@ssbarnea sorry do you mean `(using-lint-profiles)=` on line 1?

Reply:
Yeah. But if you change that one you also need to rename back to `profiles.md` too.

Reply:
+1 for leaving content creator

Reply:
@ssbarnea Yeah but `profiles.md` is generated so I thought `using-lint-profiles` would be good. I believe that would be the best choice for SEO.

"using-lint-profiles" is slightly more meaningful than "using-profiles"